Patches/NewRatkinPlus/ThingDef_Misc/Ratkin_Buildings_Security.xml

```diff
@@ -7,147 +7,222 @@
+			</li>
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[defName="BallistaBolt_Heavy"]</xpath>
+			</li>	
+			
+			<!-- ========== Ratkin Cannon ========== -->
+			  
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryBuilding_Cannon"]/thingClass</xpath>
+				<value>
+					<thingClass>CombatExtended.Building_TurretGunCE</thingClass>
+				</value>
+			</li>
+
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryBuilding_Cannon"]/inspectorTabs</xpath>
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryBuilding_Cannon"]/statBases/AccuracyTouch</xpath>
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryBuilding_Cannon"]/statBases/AccuracyShort</xpath>
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryBuilding_Cannon"]/statBases/AccuracyMedium</xpath>
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryBuilding_Cannon"]/statBases/AccuracyLong</xpath>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryBuilding_Cannon"]/building/turretBurstCooldownTime</xpath>
+				<value>
+					<turretBurstCooldownTime>1</turretBurstCooldownTime>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryWeapon_Cannon"]/building</xpath>
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryWeapon_Cannon"]/comps/li[@Class="CompProperties_ChangeableProjectile"]</xpath>
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryWeapon_Cannon"]/statBases/AccuracyTouch</xpath>
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryWeapon_Cannon"]/statBases/AccuracyShort</xpath>
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryWeapon_Cannon"]/statBases/AccuracyMedium</xpath>
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[@Name = "RK_BaseArtilleryWeapon_Cannon"]/statBases/AccuracyLong</xpath>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName = "RK_Turret_Cannon_Strait"]</xpath>
+				<value>
+				<statBases/>
+				</value>
+			</li>
+			
+			<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+				<defName>RK_Turret_Cannon_Strait</defName>
+				<statBases>
+					<Mass>20</Mass>
+					<Bulk>20</Bulk>	
+					<SightsEfficiency>1</SightsEfficiency>
+					<ShotSpread>0.12</ShotSpread>
+					<SwayFactor>1.55</SwayFactor>
+					<RangedWeapon_Cooldown>0.36</RangedWeapon_Cooldown>
+				</statBases>
+				<Properties>
+					<recoilAmount>1.13</recoilAmount>
```

Comment:
Not sure if it matters, but probably doesn't need recoil and could use the `Mounted` recoil pattern.

Reply:
Admittedly if it doesn't have recoil anyway, the mounted recoil pattern is completely irrelavant I believe. I will remove the redundant recoil stat though.